File: wcfcore5/WcfCoreWebApplication/Color.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace WcfCoreWebApplication
{
    public partial class Color
    {
        [Key]
        [Column("UID")]
        public Guid Uid { get; set; }
        [StringLength(255)]
        public string Description { get; set; }
        [Required]
        [StringLength(10)]
        public string Name { get; set; }
        public byte[] RowVersion { get; set; }
    }
}

File: emptycrud-webforms/WindnTrees.Abstraction.WebForms.App/Scripts/Model/Color.js
function Color(data) {
    var instance = this;

    instance._datakey = data.UID;

    instance.UID = ko.observable(data.UID);
    instance.Name = ko.observable(data.Name);
    instance.Description = ko.observable(data.Description);
    instance.RowVersion = data.RowVersion;
}